pkg/server/handlers/handlers_test.go

```diff
@@ -1701,27 +1713,46 @@ func getMockedControl(t *testing.T) (*config.Control, context.CancelFunc) {
 	// setting up a whole remotedialer tunnel server here
 	control.Runtime.Tunnel = http.HandlerFunc(func(resp http.ResponseWriter, req *http.Request) {})
 
+	// create a native client - note that this will not work because there is no apiserver
```

Comment:
This comment is confusing, because you proceed to keep checking for Error ToNot HaveOccurred. If this doesn't work, wouldn't we be expecting an error?